panc/src/main/java/org/quattor/pan/dml/data/ProtectedListResource.java

```diff
@@ -25,7 +25,11 @@ public Element duplicate() {
 
 	@Override
 	public Element get(Term key) throws InvalidTermException {
-		return baseList.get(key);
+		final Element value = baseList.get(key);
```

Comment:
(this is to my current understanding of the code :smile: )
so suppose you have a protected list with unprotected elements, the code below does the following: it takes the element, makes a protected copy and returns the copy. this is suboptimal. it would be better to protect the element once (but not by calling `.protect`), so the next time it doesn't have to make a copy anymore.
if the element is protected, `.protect` does not return a copy, so that is fine.